edb/pgsql/compiler/dml.py

```diff
@@ -301,6 +309,43 @@ def wrap_dml_cte(
     return dml_rvar
 
 
+def merge_iterator_scope(iterator: Optional[pgast.IteratorCTE],
+                         select: pgast.SelectStmt,
+                         *, ctx: context.CompilerContextLevel) -> None:
+    while iterator:
+        ctx.path_scope[iterator.path_id] = select
+        iterator = iterator.parent
+
+
+def merge_iterator(iterator: Optional[pgast.IteratorCTE],
```

Comment:
Please use formatting for function signatures consistent with other in the file.